tests/autoyast/verify_firewalld.pm

```diff
@@ -23,7 +23,7 @@ sub run {
     my $errors = '';
     # Verify services configured for external zone
     my $zone_config = script_output('firewall-offline-cmd --list-services --zone=public');
-    $errors .= "services are not configured for public zone, expected http and https, got: $zone_config\n" unless $zone_config =~ /http https/;
+    $errors .= "services are not configured for public zone, expected apache2 apache2-ssl, got: $zone_config\n" unless $zone_config =~ /apache2 apache2-ssl/;
```

Comment:
I assume because this file is called `verify_firewalld` it won't influence older product versions, right?

Reply:
Nope, it's scheduled in single test suite as of now, we will reuse it for firewalld style profile, but they will result in the same configuration.